### ISSUE TITLE
Nitpick: package.xml versions and tag of the repo

### ISSUE THREAD

**Issue by traversaro** [icub-tech-iit/xcub-moveit2#32]
This is really a nitpick, and we can safely ignore it for a long time, however I wanted to write down something I discovered. It seems to me that the packages versions reported in the `package.xml` files are not aligned with the ones used for tags. In particular, the last tag for the repo is 0.1.0, while the package.xml report either 0.0.0 or 0.3.0, see https://github.com/search?q=repo%3Aicub-tech-iit%2Fxcub-moveit2%20%3Cversion%3E&type=code .

However, manually updating the versions in ~6 files for each tag release of the repo seems extremely boring, so I was wondering if there was some way to that automatically, and it seems that the way to do is via https://github.com/ros-infrastructure/catkin_pkg (that despite the name is still used in ROS 2, and is a pure python package that can be installed both via pip or conda/pixi).

In particular, if I install the `catkin_pkg` conda-forge package, and run the `catkin_package_version` script in the repo, I automatically get:

~~~
(xcub-moveit2) traversaro@IITBMP014LW012:~/xcub-moveit2$ catkin_package_version
Two packages have different version numbers (0.0.0 != 0.3.0):
- ./xcub_moveit_test_controller/package.xml
- ./ergocub_moveit_config/package.xml
~~~

I align them, I get:

~~~
(xcub-moveit2) traversaro@IITBMP014LW012:~/xcub-moveit2$ catkin_package_version
0.1.0
~~~

and then I can automatically bump all the `package.xml` automatically with the ` catkin_package_version --bump` command:

~~~
(xcub-moveit2) traversaro@IITBMP014LW012:~/xcub-moveit2$ catkin_package_version
0.1.0
(xcub-moveit2) traversaro@IITBMP014LW012:~/xcub-moveit2$ catkin_package_version --bump minor
0.1.0 -> 0.2.0
(xcub-moveit2) traversaro@IITBMP014LW012:~/xcub-moveit2$ catkin_package_version
0.2.0
~~~

**Comment by Nicogene** [comment]
Maybe we can automatize the alignment of versions with a simple Github Action? @martinaxgloria @traversaro

**Comment by traversaro** [comment]
Yes, something I would like to expand also to other repo is to have a simple action with `workflow_dispatch` where you select which type of version bump you want (`major`, `minor` or `patch`) and then automatically bumps any string that is necessary to bump, and does the tag. However, in most repos the amount of stuff required to do a release are sufficiently minimal that I was never triggered enough to do that, but for this repo the situation is a bit different with the ~6 package.xml .
